examples/plot_hf_hub.py

```diff
@@ -76,7 +76,11 @@
 
 local_repo = mkdtemp(prefix="skops-")
 hub_utils.init(
-    model=pkl_name, requirements=[f"scikit-learn={sklearn.__version__}"], dst=local_repo
+    model=pkl_name,
+    requirements=[f"scikit-learn={sklearn.__version__}"],
+    dst=local_repo,
+    task="tabular-classification",
```

Comment:
Why do we need to provide task name in `hub_utils.init()` if the user already provides it in the model card?